extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.control.js

```diff
@@ -527,6 +527,14 @@ angular.module('PaperUI.controllers.control', []).controller('ControlPageControl
     if ($scope.item.state === 'UNDEF' || $scope.item.state === 'NULL') {
         $scope.item.state = '-';
     }
+    $scope.sendStateCommand = function(command) {
+        $scope.sendCommand(command);
+        if (command == "DOWN") {
+            $scope.item.state = 100;
```

Comment:
Hm, are you manually setting the state here? You should not - the runtime should send according events, which should be used to update the displayed status.